### PR TITLE
Added list component.

### PR DESCRIPTION
**Pull request by manojkumar8400**
Added different types of list in this component.

### CODE REVIEW

File: documentation/lists/lists.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522spaced-list%2522%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253EOne%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253ETwo%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253EThree%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253EFour%253C%252Fli%253E%250A%2509%253Cli%2520class%253D%2522spaced%2522%253EFive%253C%252Fli%253E%250A%253C%252Fdiv%253E"
+                        style="width: 428px; height: 322px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <hr>
+                <h2>Stacked List (Notification Page)</h2>
+
+                <div class="stacked-list">
```
Comment (Sen-442b) [inline documentation/lists/lists.html:79]: Suggestion:-   you can also add `ul`  tag as parent to the list items

**Review by Sen-442b — APPROVED** [review]
Code structure and  variable  naming looks neat and consistent to me

File: documentation/lists/lists.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                    <li class="spaced">Two</li>
+                    <li class="spaced">Three</li>
+                    <li class="spaced">Four</li>
+                    <li class="spaced">Five</li>
+                </div>
+                <div class="code-container">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522spaced-list%2522%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253EOne%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253ETwo%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253EThree%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced%2522%253EFour%253C%252Fli%253E%250A%2509%253Cli%2520class%253D%2522spaced%2522%253EFive%253C%252Fli%253E%250A%253C%252Fdiv%253E"
+                        style="width: 428px; height: 322px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (manojasarsa) [inline documentation/lists/lists.html:72]: You can make a class for iframe and define styling using it.

**Review by manojasarsa — APPROVED** [review]
Rest, code looks clean.
Good to merge

**Pull request merged**